ruleset/rules/0630-nextcloud_rules.xml

```diff
@@ -1,175 +1,163 @@
-- Examples syslog:
-- Sep  1 20:16:09 foo NextCloud[15463]: {core} Login failed: 'test' (Remote IP: '127.0.0.1')
-- Sep  1 22:16:33 foo NextCloud[15467]: {core-preview} Passed filename is not valid, might be malicious (file:"test";ip:"127.0.0.1","@source":"NextCloud")
+  Two log sources are valid for NextCloud rules:
+
+  To use the NextCloud decoder paste the following in /var/ossec/etc/ossec.conf:
+  <localfile>
+    <location>ABSOLUTE_PATH_TO_YOUR_NEXTCLOUD_.json_LOG_FILE</location>
+    <log_format>json</log_format>
+    <label key="@source">NextCloud</label>
+  </localfile>
+
+  If you enabled audit in your Nextcloud system you may use those logs instead.
+  Paste the following in /var/ossec/etc/ossec.conf:
+  <localfile>
+    <location>ABSOLUTE_PATH_TO_YOUR_AUDIT_.json_LOG_FILE</location>
+    <log_format>json</log_format>
+    <label key="@source">NextCloud</label>
+  </localfile>
 -->
 
+
+
 <group name="json,nextcloud,">
 
   <rule id="88200" level="0">
     <decoded_as>json</decoded_as>
     <field name="@source">NextCloud</field>
-    <description>NextCloud messages grouped.</description>
     <options>no_full_log</options>
+    <description>NextCloud messages grouped.</description>
   </rule>
 
   <rule id="88201" level="0">
     <decoded_as>nextcloud</decoded_as>
-    <description>NextCloud messages grouped.</description>
     <options>no_full_log</options>
+    <description>NextCloud messages grouped.</description>
   </rule>
 
   <rule id="88202" level="1">
     <if_sid>88200,88201</if_sid>
     <match>Login attempt: </match>
-    <description>NextCloud authentication attempt.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
     <options>no_full_log</options>
+    <description>NextCloud authentication attempt.</description>
+    <group>pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
   </rule>
 
-  <rule id="88210" level="6">
+  <rule id="88212" level="6">
     <if_sid>88200,88201</if_sid>
-    <match>Logout occurred</match>
-    <description>NextCloud logout successful.</description>
-    <group>pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
+    <match>Login failed: </match>
     <options>no_full_log</options>
+    <description>NextCloud authentication failed.</description>
+    <group>authentication_failed,pci_dss_10.2.4,pci_dss_10.2.5,gdpr_IV_35.7.d,gdpr_IV_32.2,hipaa_164.312.b,nist_800_53_AU.14,nist_800_53_AC.7,tsc_CC6.1,tsc_CC6.8,tsc_CC7.2,tsc_CC7.3,</group>
   </rule>
```

Comment:
Please, correct the position of the rule on the file.

Reply:
Can't do that because it's required by rule 88203. This is a legacy issue. Need to break criteria 7.a to fix this

Reply:
Literally the reason of this entire rework: https://github.com/wazuh/wazuh/pull/10476